### PR TITLE
build: add maintainers for the repo

### PR DESCRIPTION
**Pull request by sameenfatima78**
**Description:** This PR adds me and Irfan as maintainers for this repo. This repo was created by OpenCraft team as part of a blended project with our team (Enterprise Markhors).

**JIRA:** Link to JIRA ticket

**Dependencies:** dependencies on other outstanding PRs, issues, etc. 

**Merge deadline:** List merge deadline (if any)

**Installation instructions:** List any non-trivial installation 
instructions.

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [ ] tag reviewer 
- [ ] tag reviewer 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.

### CODE REVIEW

**Comment by sameenfatima78** [comment]
@feanil Hi, can you please review and merge this PR if all looks okay? The build failures are already handled in https://github.com/openedx/xblock-skill-tagging/pull/10

File: catalog-info.yaml [hunk 1]
```diff
@@ -13,20 +13,9 @@ metadata:
...
-  subcomponentOf: '<name_of_a_component>'
-
-  # (Optional) An array of different components or resources.
+  owner: group:2u-xblockskilltagging-maintainers
+  type: 'library'
+  lifecycle: 'production'
   dependsOn:
-  - '<component_or_resource>'
-  - '<another_component_or_resource>'
+  - 'lms'
```
Comment (feanil) [inline catalog-info.yaml]:
```suggestion
  - 'edx-platform'
```

unfortunately, the `lms` is not a component in backstage.

File: catalog-info.yaml [hunk 2]
```diff
@@ -13,20 +13,9 @@ metadata:
...
+  - 'taxonomy-connector'
```
Comment (feanil) [inline catalog-info.yaml:21]: This one hasn't been added to backstage yet but assuming that it has a `catalog-info.yaml` file in the future this should be fine to leave in.

**Review by feanil — APPROVED** [review]

File: catalog-info.yaml (hunk 1 above)
Comment (sameenfatima78) [reply catalog-info.yaml]: I have removed this component.

File: catalog-info.yaml (hunk 2 above)
Comment (sameenfatima78) [reply catalog-info.yaml:21]: This one also falls into our team's ownership so I'll make sure to add that file.

File: catalog-info.yaml (hunk 1 above)
Comment (sameenfatima78) [reply catalog-info.yaml]: Fixed.

**Comment by sameenfatima78** [comment]
@feanil Thanks for the review, I have incorporated your feedback. Can you please merge it?

**Review by feanil — APPROVED** [review]

**Pull request merged**